src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -117,38 +119,29 @@ Actors:
             gender: 1
             dob: 1
           name: usefulIndex
-  - Repeat: 1
-    Database: admin
-    Operations:
-    - OperationName: RunCommand
-      OperationCommand:
-        fsync: 1
   - *Nop
-  - Repeat: 1
-    Database: *DB
-    Operations:
-    - OperationName: RunCommand
-      OperationCommand:
-        drop: Collection0
+  - *Nop
 
 - Name: RunQueries
   Type: RunCommand
   Threads: 1
   Phases:
   - *Nop
   - *Nop
-  - Repeat: 10
+  - Repeat: 100
```

Comment:
Note, according to the [docs ](https://github.com/mongodb/genny/blob/86e796946f3faea4be63b33b088d0fba93543890/src/workloads/docs/HelloWorld-MultiplePhases.yml) this will run for at least 100 iterations and at least 5 minutes, whichever takes longer. Is that your intent here?

Reply:
Thank you for the note! Yes, this was my intention.